ci/code_checks.sh

```diff
@@ -140,10 +140,6 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     invgrep -R --include="*.py" --include="*.pyx" -E "(DEPRECATED|DEPRECATE|Deprecated)(:|,|\.)" pandas
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
-    MSG='Check for old-style classes' ; echo $MSG
```

Comment:
why are you removing this?

Reply:
In python2 ´´class MyClass:´´ created an different class type (old style) than ´´class MyClass(object):`` (new style). Pandas required that all classes be new style for py2/py3 compatability. This distinction has been lifted in python3, so the check isn`t needed any more.

Reply:
maybe so but we still don’t want them
i would revert the check

Reply:
I’ve added a check for the reverted case.